autoPlay.user.js

```diff
@@ -926,6 +927,13 @@ function useMetalDetectorIfRelevant() {
 	}
 }
 
+function useDecreaseCooldown(beforeAbility) {
+	if (!isAbilityCoolingDown(ABILITIES.COOLDOWN)) {
+		advLog("Decrease Cooldown before " + beforeAbility + ".", 2);
+		triggerAbility(ABILITIES.COOLDOWN);
```

Comment:
if we use it for nukes, it is ensured to be active and others also benefit. Right now we have no other usage.